PhysicsTools/PatAlgos/plugins/PATPhotonProducer.cc

```diff
@@ -370,6 +372,38 @@ void PATPhotonProducer::produce(edm::Event & iEvent, const edm::EventSetup & iSe
     aPhoton.setCryEta( ecalRegData.seedCrysEtaOrX() );
     aPhoton.setIEta( ecalRegData.seedCrysIEtaOrIX() );
     aPhoton.setIPhi( ecalRegData.seedCrysIPhiOrIY() );
+    } else {
+    aPhoton.setEMax(0);
```

Comment:
it seems more appropriate to do this in PATPhotonSlimmer rather than here.
There may be reasons to still use this data between patPhotons and slimmedPhotons and not save it in the slimmed.